.github/workflows/test.yml

```diff
@@ -27,7 +27,7 @@ jobs:
       uses: actions/cache@v2
       with:
         path: ~/audb
-        key: emodb-ubuntu
+        key: emodb-1.1.1-1
```

Comment:
why `-1`?

Reply:
Because I used `emodb-1.1.1` before and the cache contained still a wrong file, so I had to rename it again.

Reply:
Using another name is the only way to get a new clean cache as we can't delete existing once (they are automatically deleted after 7 days, but I propose not to wait until then ;) )